lib/active_record_shards/connection_switcher.rb

```diff
@@ -95,18 +95,18 @@ def on_master(&block)
     alias_method :with_slave_if, :on_slave_if
     alias_method :with_slave_unless, :on_slave_unless
 
-    def on_cx_switch_block(which, options = {}, &block)
+    def on_cx_switch_block(which, force: false, construct_ro_scope: nil, &block)
```

Comment:
You are changing the signature of a public method. This is kind of an API change.

Reply:
Options hash is equivalent to keyword arguments. But kwargs are more explicit. Not an API change imho.

Reply:
true that